src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -415,6 +415,19 @@ public FlutterCommand flutterTest(@NotNull PubRoot root, @NotNull VirtualFile fi
     return new FlutterCommand(this, root.getRoot(), FlutterCommand.Type.TEST, args.toArray(new String[]{ }));
   }
 
+  @NotNull
+  public FlutterCommand widgetPreview(@NotNull PubRoot root, boolean isVerboseMode) {
+    final List<String> args = new ArrayList<>();
+    args.add("start");
+    args.add("--web-server");
+    args.add("--machine");
```

Comment:
We should also pass `--dtd-url=<url>` here and, if the plugin hosts a DevTools instance, `--devtools-server-address=<url>`.

Does the Intellij plugin provide the DTD `Editor` service?